cpp/modmesh/view/wrap_view.cpp

```diff
@@ -160,6 +160,12 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapRApplication
                 {
                     return self.main()->pytext();
                 })
+            .def_property_readonly(
```

Comment:
Things get exposed from C++ to Python need corresponding unit tests.  At the moment I do not see the need to access it from Python.